app/views/desks/show.html.erb

```diff
@@ -1,7 +1,20 @@
-<%= link_to "Back to desk list", root_path %>
-<h3>Desk details</h3>
-<ul>
-  <li><%= @desk.id %></li>
-  <li><%= @desk.name %></li>
-  <li><%= @desk.occupied %></li>
-</ul>
+<%= stylesheet_link_tag 'show' %>
```

Comment:
Same here. Just import them in `application.scss`

Reply:
?